plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/project/ProjectUtils.java

```diff
@@ -105,24 +109,68 @@
 
     // wait for any post-import operations too
     waitUntilIdle();
+    assertFalse("imported projects have errors", projectsHaveErrors(projects));
 
     return projects;
   }
 
-  /** Wait for any spawned jobs to complete (e.g., validation jobs). */
-  public static void waitUntilIdle() {
-    while (!Job.getJobManager().isIdle()) {
-      Display display = Display.getCurrent();
-      if (display != null) {
-        while (display.readAndDispatch()) {
-          /* spin */
+  /**
+   * Return true if projects have errors; lists any found to <code>System.err</code>.
+   */
```

Comment:
Anyway to reuse [`getAllBuildErrors()`](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotProjectActions.java#L204)?

(Not related, but I also noticed that `getAllBuildErrors()` accepts `SWTWorkbenchBot` as a parameter that is not being used.)